paimon-core/src/main/java/org/apache/paimon/Snapshot.java

```diff
@@ -357,6 +362,16 @@ public static Snapshot fromJson(String json) {
     public static Snapshot fromPath(FileIO fileIO, Path path) {
         try {
             return Snapshot.fromJson(fileIO.readFileUtf8(path));
+        } catch (FileNotFoundException e) {
+            String errorMessage =
+                    String.format(
+                            "Snapshot file %s does not exist. "
+                                    + "It might have been expired by other jobs operating on this table. "
```

Comment:
"by other jobs operating on this table." -> "by other write jobs on this table." ?

Reply:
`write` may be not so accurate, maybe just expire procedure.

Reply:
OK